databricks/koalas/series.py

```diff
@@ -4518,6 +4519,87 @@ def repeat(self, repeats: int) -> "Series":
         else:
             return _col(ks.concat([kdf] * repeats))
 
+    def asof(self, where, subset=None):
```

Comment:
@itholic can we deduplicate the implementation this with `Index.asof`? We could do it in `IndexOpsMixin`

Reply:
Yes we can duplicate them, but I think there are more confusions than advantages because of the several reasons like the below.

- the name of parameter between `Index.asof` and `Series.asof` is different (`where` for `Series.asof`, `label` for `Index.asof`)
- And the acceptable types of those parameters are also different. (`labels` can only accept scalar type whereas `where` can accept scalar, pd.Index, pd.Series, list, tuples, dict... )

So I think the type validation logic could be too complicated.

What do you think ??

Reply:
Okay, let's leave it separate.